ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/SigningEthSendTransactionIntegrationTest.java

```diff
@@ -341,4 +344,23 @@ public void transactionWithMissingNonceReturnsErrorsOtherThanLowNonceToCaller()
     sendRequestThenVerifyResponse(
         request.ethSigner(sendTransaction.missingNonce()), response.ethSigner(INVALID_PARAMS));
   }
+
+  @Test
+  public void moreThanFiveNonceTooLowErrorsReturnsAnErrorToUser() {
+    setupEthNodeResponse(".*eth_sendRawTransaction.*", response.ethNode(NONCE_TOO_LOW), 6);
+
+    sendRequestThenVerifyResponse(
+        request.ethSigner(sendTransaction.missingNonce()), response.ethSigner(INTERNAL_ERROR));
```

Comment:
Consistency; either reference the enum `HttpResponseStatus` (as you've added below) or change that reference to use static import too

Reply:
Changed below to use static import - but not sure that covers the issue?

Reply:
✔️